modules/ROOT/pages/keycloak/coding.adoc

```diff
@@ -90,4 +139,57 @@ try! AgsAuth.instance.configure(authConfig: authenticationConfig)
 .Cordova
 ****
 
-****
\ No newline at end of file
+Import and initialize Auth:
+
+[source,javascript]
+----
+const authService = new Auth();
+const initOptions = { onLoad: "login-required" };
+
+authService.init(initOptions)
+    .then(() => {
+        // successful init & authentication
+    })
+    .catch((err) => {
+        // initialization error
+    });
+----
+
+You can pass `login-required` or `check-sso` to the init function. `login-required` will authenticate the client if the user is logged in to Keycloak or display the login page if not. `check-sso` will only authenticate the client if the user is already logged in. If the user is not logged in the browser will be redirected back to the application and remain unauthenticated. By default, the `check-sso` option is used.
+
+NOTE: Initialization will also perform authentication
+
+****
+
+[role="secondary"]
+.Xamarin
+****
+*Initializing Auth in an Android app (MainActivity.cs)*
+
+[source,csharp]
+----
+var app = new App();
+MobileCoreAndroid.Init(app.GetType().Assembly,ApplicationContext);
+var authService = AuthService.InitializeService();
+var authConfig = AuthenticationConfig.Builder.RedirectUri("org.aerogear.mobile.example:/callback").Build();
+authService.Configure(authConfig);
+----
+
+NOTE: For Android an link:https://developer.android.com/guide/topics/manifest/intent-filter-element[intent-filter] should be configured with the callback URL specified in AuthenticateOptions in the apps AndroidManifest.xml. See the link:https://github.com/aerogear/xamarin-showcase-template/blob/master/Example.Android/Properties/AndroidManifest.xml[example app].
```

Comment:
- `Intent filter` is more appropriate, also link to https://developer.android.com/guide/topics/manifest/manifest-intro#ifs
- I think `apps AndroidManifest.xml` -> `app's AndroidManifest.xml`.
- Let's add this intent filter requirement as a step instead of pointing to the example app, if it's necessary.